pubspec.yaml

```
@@ -10,7 +10,7 @@ dependencies:
   flutter:
     sdk: flutter
 
-  intl: ^0.17.0
+  intl: 0.18.0
```

May be you should put the `^`in of the version to ensure that it will work in future versions.
```suggestion
  intl: ^0.18.0
```